docs/deployment/README.md

```diff
@@ -525,43 +499,34 @@ You should be able to visit the dashboard website at `http://<prefix>-dashboard.
 
 ## Updating Cumulus deployment
 
-Once deployed for the first time, any future updates to the role/stack configuration files/version of Cumulus can be deployed and will update the appropriate portions of the stack as needed.
+Once deployed for the first time, any future updates to the configuration files, Terraform files (`*.tf`), or version of Cumulus can be deployed and will update the appropriate portions of the stack as needed.
 
 ## Cumulus Versioning
 
 Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (i.e. 1.0.0). We use Lerna to manage versioning.
 
-## Update roles
+## Update data persistence resources
 
-```bash
-  $ DEPLOYMENT=<deployment-name> \
-      AWS_REGION=<region> \ # e.g. us-east-1
-      AWS_PROFILE=<profile> \
-      npm run deploy-iam
-```
-
-## Update database
+From your `data-persistence-tf` directory:
 
 ```bash
-  $ DEPLOYMENT=<deployment-name> \
-      AWS_REGION=<region> \ # e.g. us-east-1
+  $ AWS_REGION=<region> \ # e.g. us-east-1
       AWS_PROFILE=<profile> \
-      npm run deploy-db
+      terraform apply
 ```
 
 ## Update Cumulus
 
+From your `cumulus-tf` directory:
+
 ```bash
-  $ DEPLOYMENT=<deployment-name> \
-      AWS_REGION=<region> \ # e.g. us-east-1
+  $ AWS_REGION=<region> \ # e.g. us-east-1
       AWS_PROFILE=<profile> \
-      npm run deploy-app
+      terraform apply
 ```
 
```

Comment:
Not sure that it belongs at this position in the README, but I feel like we ought to either cross reference (our future currently non existant) docs or included a section on  why losing your state is bad, etc.   Possibly L160?

Reply:
Just to say that if your state is gone/corrupt, then `terrraform apply` will have unpredictable results or more likely not work altogether?

Reply:
236865254260ac15a6973d70b26a9f94227a6e3d